announcements/1.16.3.md

```diff
@@ -0,0 +1,25 @@
+### Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.16.3.
```

Comment:
Should we clarify that the update is only for the appliance (but that there is an action required for the full deployment)?

Reply:
That there is an action required for the full deployment is not clear. We don't provide Grafana with it. So all we can do is give a hint at the end that if someone has Grafana deployed, they need to update the key.

Reply:
How about:
```suggestion
The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.16.3. This release updates only the Anbox Cloud Appliance. However, if you run the regular Anbox Cloud and have Grafana deployed, an action is required (see below).
```